dashboard/test/controllers/pd/workshop_enrollment_controller_test.rb

```diff
@@ -30,6 +30,20 @@ class Pd::WorkshopEnrollmentControllerTest < ::ActionController::TestCase
     )
   end
 
+  test 'non-logged-in users can not enroll in csd' do
+    workshop = create :pd_workshop, course: Pd::Workshop::COURSE_CSD
+    get :new, params: {workshop_id: workshop.id}
+    assert_response :success
+    assert_template :logged_out
+  end
+
+  test 'non-logged-in users can not enroll in csp' do
+    workshop = create :pd_workshop, course: Pd::Workshop::COURSE_CSP
+    get :new, params: {workshop_id: workshop.id}
+    assert_response :success
+    assert_template :logged_out
+  end
+
   test 'non-logged-in users can enroll' do
```

Comment:
can we update this test name to specify that this case only covers CSF workshops?

Reply:
And can we also explicitly test enrolling a logged-in user in a csd or csp workshop?